bindings/java/src/main/native/bridge.cc

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#include "bridge.h"
+#include "request_util.h"
+
+extern "C" {
```

Comment:
this looks like C code, why need to compile it as C++?

Reply:
jni code is cleaner when written in C++

Reply:
i guess you mean the code which will be in next PR?

Reply:
Yes

Reply:
no need to use extern "C" anymore, @hoopoepg  just added it to UCX haders